generators/app/templates/src/app/screens/OnBoarding/components/Swiper/components/Footer/buttonsInfo.js

```diff
@@ -0,0 +1,35 @@
+import i18next from 'i18next';
+
+export const screensButtonInfo = ({ nextScreen, skip, previousScreen, screenName }) =>
+  ({
+    FirstScreen: {
+      firstButton: {
+        onPress: skip,
+        title: i18next.t('ONBOARDING:SKIP')
+      },
+      secondButton: {
+        onPress: nextScreen,
+        title: i18next.t('ONBOARDING:NEXT')
+      }
+    },
+    SecondScreen: {
+      firstButton: {
+        onPress: previousScreen,
+        title: i18next.t('ONBOARDING:PREVIOUS')
+      },
+      secondButton: {
+        onPress: nextScreen,
+        title: i18next.t('ONBOARDING:NEXT')
+      }
+    },
+    ThirdScreen: {
```

Comment:
is it possible to use an array instead of an object here? and you use the `index` value of the swiper to get the correct screen.